core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -21,6 +21,46 @@
     html, body {
       background-color: #eee;
     }
+
+    /* TODO(mgowano): Implement horizontal scrolling using Javascript. */
```

Comment:
Oh I am going to have this done in a separate PR. I think for now normal scrolling using css should be fine.